apps/etterlatte-behandling/src/main/kotlin/config/ApplicationContext.kt

```diff
@@ -0,0 +1,192 @@
+import java.time.temporal.ChronoUnit
+
+private fun pdlHttpClient(config: Config) = httpClientClientCredentials(
+    azureAppClientId = config.getString("azure.app.client.id"),
+    azureAppJwk = config.getString("azure.app.jwk"),
+    azureAppWellKnownUrl = config.getString("azure.app.well.known.url"),
+    azureAppScope = config.getString("pdl.azure.scope")
+)
+
+private fun grunnlagHttpClient(config: Config) = httpClientClientCredentials(
+    azureAppClientId = config.getString("azure.app.client.id"),
+    azureAppJwk = config.getString("azure.app.jwk"),
+    azureAppWellKnownUrl = config.getString("azure.app.well.known.url"),
+    azureAppScope = config.getString("grunnlag.azure.scope")
+)
+
+private fun navAnsattHttpClient(config: Config) = httpClientClientCredentials(
+    azureAppClientId = config.getString("azure.app.client.id"),
+    azureAppJwk = config.getString("azure.app.jwk"),
+    azureAppWellKnownUrl = config.getString("azure.app.well.known.url"),
+    azureAppScope = config.getString("navansatt.azure.scope")
+)
+
+private fun featureToggleProperties(config: Config) = mapOf(
+    FeatureToggleServiceProperties.ENABLED.navn to config.getString("funksjonsbrytere.enabled"),
+    FeatureToggleServiceProperties.APPLICATIONNAME.navn to config.getString(
+        "funksjonsbrytere.unleash.applicationName"
+    ),
+    FeatureToggleServiceProperties.URI.navn to config.getString("funksjonsbrytere.unleash.uri"),
+    FeatureToggleServiceProperties.CLUSTER.navn to config.getString("funksjonsbrytere.unleash.cluster")
+)
+
+class ApplicationContext(
+    val env: Map<String, String> = System.getenv(),
+    val config: Config = ConfigFactory.load(),
+    val rapid: KafkaProdusent<String, String> = GcpKafkaConfig.fromEnv(env)
+        .standardProducer(env.getValue("KAFKA_RAPID_TOPIC")),
+    val featureToggleService: FeatureToggleService = FeatureToggleService.initialiser(featureToggleProperties(config)),
+    val pdlHttpClient: HttpClient = pdlHttpClient(config),
+    val grunnlagHttpClient: HttpClient = grunnlagHttpClient(config),
+    val navAnsattKlient: NavAnsattKlient = NavAnsattKlientImpl(
+        navAnsattHttpClient(config),
+        env.getValue("NAVANSATT_URL")
+    ),
+    val norg2Klient: Norg2Klient = Norg2KlientImpl(httpClient(), env.getValue("NORG2_URL")),
+    val leaderElectionHttpClient: HttpClient = httpClient(),
+    val vedtakKlientObo: VedtakKlient = VedtakKlientImpl(config, httpClient()),
+    val grunnlagKlientObo: GrunnlagKlient = GrunnlagKlientObo(config, httpClient())
+) {
+    val getSaksbehandlerGroupIdsByKey =
+        run {
+            val attestantClaim = env["AZUREAD_ATTESTANT_GROUPID"]!!
+            val saksbehandlerClaim = env["AZUREAD_SAKSBEHANDLER_GROUPID"]!!
+            val strengFortroligClaim = env["AZUREAD_STRENGT_FORTROLIG_GROUPID"]!!
+            val fortroligClaim = env["AZUREAD_FORTROLIG_GROUPID"]!!
+            val egenAnsattClaim = env["AZUREAD_EGEN_ANSATT_GROUPID"]!!
```

Comment:
Her kan `requireEnvValue` brukes for klarere feilmelding hvis verdien ikke finnes

Reply:
Ser at Chris har gjort en del endringer på det der i main nå - men rebaser på det så bør det løses 👍

Reply:
Fikset i main nå